app/src/main/java/io/homeassistant/companion/android/webview/WebViewActivity.kt

```diff
@@ -386,6 +387,16 @@ class WebViewActivity : BaseActivity(), io.homeassistant.companion.android.webvi
                     )
                 }
 
+                override fun onRenderProcessGone(
+                    view: WebView?,
+                    handler: RenderProcessGoneDetail?
+                ): Boolean {
+                    Log.e(TAG, "onRenderProcessGone: webView crashed")
+                    view!!.reload()
```

Comment:
This will crash if `view` is `null` we should avoid crashing when possible

Reply:
confirm

Reply:
Reading the description again I dont think WebView can load in certain cases which is why the description says to remove the view from the hierarchy

`The application's implementation of this callback should only attempt to clean up the specific WebView given as a parameter, and should not assume that other WebView instances are affected. The given WebView can't be used, and should be removed from the view hierarchy, all references to it should be cleaned up, e.g any references in the Activity or other classes saved using [View.findViewById(int)](https://developer.android.com/reference/android/view/View#findViewById(int)) and similar calls, etc.`

have you tested this fix against the site Google recommended to test against? 

` To cause an render process crash for test purpose, the application can call loadUrl("chrome://crash") on the WebView.`